<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Visualization</class>
 <widget class="QWidget" name="Visualization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1373</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualization</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,2,3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="buttonLoadResult">
           <property name="text">
            <string>Load Result</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonClearResult">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditResultPath"/>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditDescription"/>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSaveFigure">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save Figure</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Show:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboSheet"/>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableClinicalStatistic"/>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditModelDescription">
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>Survival Prediction</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEditorRefDescription">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>200</height>
            </size>
           </property>
           <property name="lineWrapMode">
            <enum>QTextEdit::NoWrap</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,0">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QTableWidget" name="tableRefData">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Model</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboSurvivalModel"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QRadioButton" name="radioSurvivalSplitDataset">
                   <property name="text">
                    <string>Split By DataSet</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">buttonGroup</string>
                   </attribute>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioSurvivalSplitFeature">
                   <property name="text">
                    <string>Split By Feature</string>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">buttonGroup</string>
                   </attribute>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QCheckBox" name="checkSurvivalKM">
                 <property name="text">
                  <string>KM Result</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QCheckBox" name="checkSurvivalTrain">
                   <property name="text">
                    <string>Train</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkSurvivalCvVal">
                   <property name="text">
                    <string>CV Val</string>
                   </property>
                   <property name="checked">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkSurvivalTest">
                   <property name="text">
                    <string>Test</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QPushButton" name="buttonLoadRefData">
                   <property name="text">
                    <string>Ref Data Load</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <widget class="QLabel" name="label_10">
                     <property name="text">
                      <string>Split Feature</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="comboRefFeature"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLineEdit" name="lineEditRefSplit"/>
                   </item>
                   <item>
                    <widget class="QPushButton" name="buttonSplitShow">
                     <property name="text">
                      <string>Split Show</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="MatplotlibWidget" name="canvasSurvival" native="true">
             <property name="minimumSize">
              <size>
               <width>600</width>
               <height>600</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>Model Visualization</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MatplotlibWidget" name="canvasFeature" native="true">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="1,1">
           <item row="1" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,5">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>20</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Model</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboModelContribution"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>20</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Stretch Window</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="spinCoefficientBias">
                 <property name="minimum">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>0.900000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.050000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="radioContribution">
             <property name="text">
              <string>Feature Contribution</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_2</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QCheckBox" name="checkCindexCV">
                 <property name="text">
                  <string>CV Val</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkCindexTrain">
                 <property name="text">
                  <string>Train</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkCindexTest">
                 <property name="text">
                  <string>Test</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,5">
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>Model</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboCindexModel"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioVariance">
             <property name="text">
              <string>C-index Variance</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_2</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header location="global">MatplotlibWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
